day6/salestax.py

```diff
@@ -1,33 +1,68 @@
-import csv
-import pandas
 import fs 
+import csv
+import json
+
+import pandas as pd
+
+class calculation_of_tax:
+
+    def read_File(self):
```

Comment:
- it should be `read_file`, not `read_File`
- take `file_path` and `delimiter` as input to the method.